expression/builtin_json.go

```diff
@@ -216,11 +216,14 @@ func (c *jsonUnquoteFunctionClass) getFunction(ctx sessionctx.Context, args []Ex
 	return sig, nil
 }
 
-func (b *builtinJSONUnquoteSig) evalString(row chunk.Row) (string, bool, error) {
+func (b *builtinJSONUnquoteSig) evalString(row chunk.Row) (res string, isNull bool, err error) {
 	str, isNull, err := b.args[0].EvalString(b.ctx, row)
 	if isNull || err != nil {
 		return "", isNull, err
 	}
+	if len(str) >= 2 && str[0] == '"' && str[len(str)-1] == '"' && !goJSON.Valid([]byte(str)) {
+		return "", false, json.ErrInvalidJSONText.GenWithStackByArgs("The document root must not be followed by other values.")
```

Comment:
Looks like still not compatible with MySQL. Maybe should add an integrated test.

    drop table if exists t1;
    create table t1(c1 varchar(100));
    insert into t1 values('""abc""');
    insert into t1 values('""abc""');
    insert into t1 values('""abc""');
    select json_unquote(c1) from t1;
    mysql> select json_unquote(c1) from t1;
    +------------------+
    | json_unquote(c1) |
    +------------------+
    | "abc"            |
    | "abc"            |
    | "abc"            |
    +------------------+
    3 rows in set (0.01 sec)

Reply:
fixed

Reply:
OK. Please complete the corresponding modification of COPR in time. Otherwise the behavior will not be compatible.